spec/features/saved_jobs/user_can_save_job_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.feature "Registered user can save a job for later" do
+  it "should add job and see it in saved jobs" do
+    company1= create_company
+    job= Job.create(title:"Programmer", description: "test", salary:"10,000",company_id: company1.id)
+    user = User.create(username: "lane", email: "lane@nil.com", password: "password")
+    visit login_path
+
+    fill_in "Username", with: "lane"
+    fill_in "Password", with: "password"
+    click_button "Login"
+
+    visit jobs_path
+
+    click_button("Save For Later")
```

Comment:
@NicoleKMarino we can add an assertion that to test that the count of saved jobs has incremented by one.:
it { should change { SavedJobs.count }.by 1 }